src/commands/create.ts

```diff
@@ -57,22 +80,28 @@ class CreateCommand extends ApifyCommand {
 
         if (!templateArchiveUrl) {
             const templateDefinition = await getTemplateDefinition(templateName, manifestPromise);
-            ({ archiveUrl: templateArchiveUrl, skipOptionalDeps, messages } = templateDefinition);
+            ({ archiveUrl: templateArchiveUrl, messages } = templateDefinition);
             this.telemetryData.templateId = templateDefinition.id;
             this.telemetryData.templateName = templateDefinition.name;
             this.telemetryData.templateLanguage = templateDefinition.category;
+
+            // This "exists"
+            if ('skipOptionalDeps' in templateDefinition) {
+                skipOptionalDeps = templateDefinition.skipOptionalDeps as boolean;
+            }
         }
 
         const cwd = process.cwd();
-        const actFolderDir = path.join(cwd, actorName);
+        const actFolderDir = join(cwd, actorName);
 
         // Create actor directory structure
         try {
-            fs.mkdirSync(actFolderDir);
-        } catch (err) {
+            mkdirSync(actFolderDir);
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        } catch (err: any) {
             if (err?.code === 'EEXIST') {
-                outputs.error(`Cannot create new Actor, directory '${actorName}' already exists. `
-                    + 'You can use "apify init" to create a local Actor environment inside an existing directory.');
+                error(`Cannot create new actor, directory '${actorName}' already exists. `
```

Comment:
this looks like a wrong rebase, we want the upper cased variant, ideally try to check other places where we have lower cased `actor` as that is almost always wrong